node/malus/integrationtests/0001-dispute-valid-block.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Disputes
+
+  Scenario: Dispute Valid Block
+    Given a test network
+    Then alice is up
+    And alice reports node_roles is 4
+    And alice reports sub_libp2p_is_major_syncing is 0
+    Then sleep 15 seconds
+    Then alice reports block height is greater than 2
+    And alice reports peers count is at least 2
+    Then bob is up
+    And bob reports block height is greater than 2
+    And bob reports peers count is at least 2
+    Then charlie is up
+    And charlie reports block height is greater than 2
+    And charlie reports peers count is at least 2
+    Then david is up
+    And alice reports polkadot_parachain_candidate_open_disputes is 1
+    And bob reports polkadot_parachain_candidate_open_disputes is 1
+    And charlie reports polkadot_parachain_candidate_open_disputes is 1
+    Then alice polkadot_parachain_candidate_dispute_votes is at least 1
+    And bob polkadot_parachain_candidate_dispute_votes is is at least 2
+    And charlie polkadot_parachain_candidate_dispute_votes is at least 3
+    Then alice polkadot_parachain_candidate_dispute_concluded is "valid"
+    And bob polkadot_parachain_candidate_dispute_concluded is "valid"
+    And charlie polkadot_parachain_candidate_dispute_concluded is "valid"
```

Comment:
We need to add a metric for that, everything here is based on prometheus metrics.

Reply:
IIUC, it should be possible to query RPC via a JS snippet (and w/o JS after metadata PR is in), here is an example:
https://github.com/paritytech/polkadot/blob/master/simnet_tests/tests/parachains.feature

However, this can be implemented as a follow-up.

Reply:
I wanted to avoid all the js query parts as much as possible, it makes everything super messy and hard to maintain. My preferred path would be, to catch as much as possible with metrics and then eventually integrate rpc query features directly into `gurke`.

Reply:
How would you have a metric for a specific candidate being included in the finalized chain? Metrics are statistical and this is a specific event